FPSCamera/Cam/FollowCam.cs

```diff
@@ -63,9 +68,19 @@ public override bool Validate()
         public override Utils.Infos GetTargetInfos() => _target.GetInfos();
 
         public override Positioning GetPositioning()
-            => _target.GetPositioning().Apply(_LocalOffset)
-                                       .Apply(Config.G.FollowCamOffset.AsOffSet)
-                                       .Apply(_inputOffset);
+        {
+            var pos = _target.GetPositioning();
+            var look = GetSmoothLookPos();
```

Comment:
this line is similar to `GetPositioning()` which calls `vehicle.GetSmoothPosition()`.